Bio/AlignIO/MafIO.py

```diff
@@ -35,14 +35,7 @@
 import os
 from itertools import islice
 
-try:
-    from sqlite3 import dbapi2 as _sqlite
-except ImportError:
-    # Not present on Jython, but should be included in Python 2.5
-    # or later (unless compiled from source without its dependencies)
-    # Still want to offer simple parsing/output
-    _sqlite = None
```

Comment:
I've actually had this on a self compiled Python, but that is a rare corner case - and it does complicate the test suite too.

Reply:
The thing is that in this module, there is no check if `_sqlite` is `None`. Then there is no point of setting it to `None` here.

Reply:
I'm thinking of the main SeqIO and SearchIO indexing tests, which are skipped when SQLite is missing.